### DISCUSSION TITLE
My code raises "httpx.connecttimeout: timed out" exception.

### DISCUSSION THREAD

**Discussion by StegaDP** [encode/httpx#2568] [Q&A]
Hey there! Today I was prepairing my project to start on Debian11 server. I'm near level to Junior Python Developer and I don't have any ideas what to do with that. Actually, you could say that I'm just stupid, but my code is working on my Windows Laptop. And exception raises only on Debian server. Thanks for your help!
![image](https://user-images.githubusercontent.com/102144407/213747650-bc44ca4b-564a-4a97-89fd-ae5e876b8b8f.png)

**Comment by smallfish06** [comment]
Does it always raised? if so, ping to target server
